nucypher/config/keys.py

```diff
@@ -15,7 +16,7 @@
 from web3.auto import w3
 
 from nucypher.config import utils
-from nucypher.config.configs import _DEFAULT_CONFIGURATION_DIR
+from nucypher.config.configs import _DEFAULT_CONFIGURATION_DIR, NucypherConfiguration
```

Comment:
Is `nucypher.config.configs` coming in a later PR? It seems it doesn't exist right now

Reply:
Hmm, ultimately its been deprecated.  Looks like this is cruft from the refactoring.

Reply:
I checked the top of #382 and it seems that `nucypher/config/configs.py` is still gone there, so it seems `nucypher/config/keys.py` it's outdated?

Reply:
Yes - as mentioned, `configs.py` is ultimately deprecated; however config/keys.py does exist.

Reply:
It looks like the `config.keys.py` module has is not being executed in automated testing! 

Good catch